src/check_jsonschema/builtin_schemas/vendor/readthedocs.json

```diff
@@ -152,9 +152,11 @@
                 "3.10",
                 "3.11",
                 "3.12",
+                "3.13",
```

Comment:
@sirosen wdyt about making a patch release with the changes here? I'm particularly interested in this new `3.13` option in RTD?

Reply:
Yeah, I'll take care of it this evening! Sorry about the delay. I've let this sit too long.

Reply:
No worries, thanks!